tests/e2e/specs/shopper/cart-proceed-to-checkout.test.js

```diff
@@ -0,0 +1,40 @@
+/**
+ * Internal dependencies
+ */
+import { shopper } from '../../../utils';
+import { SIMPLE_PRODUCT_NAME } from '../../../utils/constants';
+
+const block = {
+	name: 'Cart',
+};
+
+if ( process.env.WOOCOMMERCE_BLOCKS_PHASE < 2 )
+	// eslint-disable-next-line jest/no-focused-tests
+	test.only( `skipping ${ block.name } tests`, () => {} );
+
+describe( 'Shopper → Cart → Can proceed to checkout', () => {
+	beforeAll( async () => {
+		await shopper.block.emptyCart();
+	} );
+
+	afterAll( async () => {
+		await shopper.block.emptyCart();
+	} );
+
+	it( 'allows customer to proceed to checkout', async () => {
+		await shopper.goToShop();
+		await shopper.addToCartFromShopPage( SIMPLE_PRODUCT_NAME );
+		await shopper.block.goToCart();
+
+		// Click on "Proceed to Checkout" button
+		await Promise.all( [
```

Comment:
You could extract this into a `shopper.block.proceedToCheckout()` function as I think I wrote something similar for my test, and will likely be used by other tests

Reply:
This is a great idea, @alexflorisca! Thank you, I'll apply this on the next PR since I did go ahead and merge this one!